website/pages/use-cases/network-infrastructure-automation.jsx

```diff
@@ -78,7 +118,7 @@ export default function NetworkInfrastructureAutomationPage() {
               {
                 text: 'Learn More',
                 url:
-                  'https://learn.hashicorp.com/consul?track=integrations#integrations',
+                  '/docs/nia',
```

Comment:
We link to the nia tech preview page from the `Reduce Downtime and Outages` feature?  This feels wrong, but I'm not 100% positive here. It's not TOO far off from what this link used to be though.

@pcmccarron, can you verify this is intentional?

![CleanShot 2020-10-12 at 19 29 48@2x](https://user-images.githubusercontent.com/2105067/95808456-50473f00-0cc1-11eb-902c-5744ac1334af.png)

----->


![CleanShot 2020-10-12 at 19 36 56@2x](https://user-images.githubusercontent.com/2105067/95808900-538efa80-0cc2-11eb-855b-b0fa3cd18b73.png)